cassandra2/pom.xml

```diff
@@ -78,4 +86,60 @@ LICENSE file.
       </properties>
     </profile>
   </profiles>
+  <!-- sigar-dist can be downloaded from jboss repository -->
+	<repositories>
+		<repository>
+			<id>central2</id>
+			<name>sigar Repository</name>
+			<url>http://repository.jboss.org/nexus/content/groups/public-jboss/</url>
+			<layout>default</layout>
+			<snapshots>
+				<enabled>false</enabled>
+			</snapshots>
+		</repository>
+	</repositories>
+	<!-- unzip sigar-dist/lib files. 
+		References: 
+		http://stackoverflow.com/questions/5388661/unzip-dependency-in-maven 
+		https://arviarya.wordpress.com/2013/09/22/sigar-access-operating-system-and-hardware-level-information/
+	 -->
+ 	<build>
+	 	<plugins>
+	 		<plugin>
+			    <groupId>org.apache.maven.plugins</groupId>
+			    <artifactId>maven-dependency-plugin</artifactId>
+			    <executions>
+			      <execution>
+			        <id>unpack-sigar</id>
+			        <phase>process-test-resources<!-- or any other valid maven phase --></phase>
+			        <goals>
+			          <goal>unpack-dependencies</goal>
+			        </goals>
+			        <configuration>
+			          <includeGroupIds>org.hyperic</includeGroupIds>
+			          <includeArtifactIds>sigar-dist</includeArtifactIds>
+			          <includes>**/sigar-bin/lib/*</includes>
+					  <excludes>**/sigar-bin/lib/*jar</excludes>
+			          <outputDirectory>
+			             ${project.build.directory}/cassandra-dependency
+			          </outputDirectory>
+			        </configuration>
+			      </execution>
+			    </executions>
+			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-surefire-plugin</artifactId>
+				<version>2.8</version>
+				<configuration>
+				<systemProperties>
+				<property>
+					<name>java.library.path</name>
```

Comment:
surefire issues a warning that java.library.path can't be set this way and we need to use the argLine configuration value.